wsrpc.es6.js

```diff
@@ -61,6 +61,24 @@ function getAbsoluteWsUrl(url) {
 	return `${scheme}//${host}${port}/${path}`;
 }
 
+function buildProxyMethodHandler(property){
+  return {
+    get: (handler, childProperty) => {
+      if (childProperty in handler) {
+        return handler[childProperty];
```

Comment:
this need to avoid max depth recursion by browser utils, because Proxy is over function I don't think that it's problem